runtime/vm/JVMImage.cpp

```diff
@@ -23,14 +23,12 @@
 #include "JVMImage.hpp"
 
 JVMImage *JVMImage::_jvmInstance = NULL;
```

Comment:
Should have mentioned this earlier. We typically dont use non-const static variables because we support (or use to support) creating multiple JVMs in the same process. This is why we stick many variables off of `J9JavaVM`. You can think of`J9JavaVM` as a global static structure

Reply:
Ok I can put the PR in later for this. Plan is to have a void * jvmInstance inside of JavaVM struct and then take it from there. I looked at code and it looks like all the exposed functions (extern c functions) could take vm. At least everywhere they would be called from has access to it. Would this work?

Reply:
well except the mem_allocate functions which only take omrportlibrary but there might be a way we could redirect them to a function that requires vm. For now I am going to close the comment but we will come with a way to fix this structure.

Reply:
Wait going back to the first comment. If it is a global static struct is there a way I can access the VM without it being passed in.

Reply:
There is a way but I would not advice that. If you look at most of the functions in `vm_api.h` they usually pass in `J9VMThread` (which you can do `vmthread->javaVM`) or `J9JavaVM`. So anywhere you need to use the JVMImage functions youll have access to the `J9JavaVM`